operate/client/src/App/ProcessInstance/ProcessInstanceHeader/ProcessInstanceOperations.tsx

```diff
@@ -40,6 +41,22 @@ const ProcessInstanceOperations: React.FC<Props> = ({processInstance}) => {
   ] = useState(false);
 
   const {data: hasActiveOperationLegacy} = useHasActiveOperations();
+  const [isV1ResolveIncidentPending, setIsV1ResolveIncidentPending] =
+    useState(false);
+
+  const {data: operationsData} = useOperations();
+
+  useEffect(() => {
+    if (
+      !operationsData?.some(
+        (operation) =>
+          operation.type === 'RESOLVE_INCIDENT' &&
+          ACTIVE_OPERATION_STATES.includes(operation.state),
+      )
+    ) {
+      setIsV1ResolveIncidentPending(false);
+    }
+  }, [operationsData]);
```

Comment:
Let's at least add a comment with an issue that this needs to be removed with a mutation later when we have a proper solution